test/clj/cider/nrepl/middleware/info_test.clj

```diff
@@ -56,7 +69,7 @@
 
   (is (not (info/info-clj 'clojure.core (gensym "non-existing"))))
   (is (not (info/info-clj 'cider.nrepl.middleware.info-test (gensym "non-existing")))
-      "Check that deftype T (which returns nil for .getPackage), doesn't throw")
+      "Check that deftype T (which returns nil for .getPackage), doesn't throw") ;; TODO: ???
```

Comment:
The test doesn't seem to do what the description says. It looks the test tries to resolve a function that doesn't exist in the current namespace -- I can't see anywhere where it tries to resolve the protocol T. (I'm still trying to learn about Protocols/Records/Types, so I wasn't sure if I'm missing something here)